rules/do-you-know-how-to-delete-orphaned-users-from-the-people-picker/rule.md

```diff
@@ -11,20 +11,19 @@ redirects: []
 
 ---
 
-The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory. 
+The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory.
 <!--endintro-->
-![](PeoplePicker.jpg)
-**Figure: The People Picker** 
+![Figure: The People Picker](PeoplePicker.jpg)
+**Figure: The People Picker**
```

Comment:
Hey @tombui99 

You should delete the duplicated captions (text in bold)

- [ ] 1. Please action

Reply:
Hey @tiagov8 

Done - deleted duplicated captions